values-elasticsearch.yaml

```diff
@@ -1,13 +1,15 @@
 replicas: 1
 minimumMasterNodes: 1
 
+esJavaOpts: "-Xmx512m -Xms512m"
+
 resources:
   requests:
     cpu: "100m"
-    memory: "2Gi"
+    memory: "1Gi"
   limits:
     cpu: "1000m"
-    memory: "2Gi"
+    memory: "1Gi"
```

Comment:
I was actually playing around with this as well and wanted to verify a bit more before suggesting.
Just FYI: reducing to 500Mi (and heap: 256Mi) caused unexpected restarts. The 1Gi setting you propose here seems stable for now